observium/Dockerfile

```diff
@@ -113,7 +114,7 @@ COPY observium.conf /etc/syslog-ng/conf.d/observium.conf
 EXPOSE 514/udp
 EXPOSE 8668/tcp
 
-VOLUME ["/config","/opt/observium/logs","/opt/observium/rrd"]
+VOLUME ["/config","/opt/observium/logs","/opt/observium/rrd","/opt/observium/certificates"]
```

Comment:
I'm not sure if it should need another volume mapping just for the certificates.

Ideally, it should use Let's Encrypt's Certbot to automatically provision a certificate. DNS-01 challenges could make sense since I don't have my Observium exposed to the Internet.

Reply:
That's just how I had setup my environment. A separate directory is not necessary. I manage my own internal CA and do not expose Observium to the internet. The DNS-01 challenges from LetEncrypt should certainly work.